validator/jit_machine.go

```diff
@@ -67,7 +67,7 @@ func (machine *JitMachine) prove(
 	state := GoGlobalState{}
 
 	timeout := time.Now().Add(60 * time.Second)
-	tcp, err := net.ListenTCP("tcp", &net.TCPAddr{})
+	tcp, err := net.ListenTCP("tcp4", &net.TCPAddr{})
```

Comment:
> If the IP field of laddr is nil or an unspecified IP address, ListenTCP listens on all available unicast and anycast IP addresses of the local system

One other note here, cc @rachel-bousfield do we need to explicitly limit this to listening on localhost? I think so.